File: FormalBench-data/FilteredData/translated_c/seed_726_per_category_10_653ade686f/ParabolaVertex.c
#include "java_arrays.h"
#include <math.h>
#include <stdint.h>

JDoubleArray parabolaVertex(int32_t a, int32_t b, int32_t c) {
    JDoubleArray vertex = jdouble_array_new(2);
    double x = -(double)b / (2.0 * (double)a);
    double four_a_c = 4.0 * (double)a;
    four_a_c = four_a_c * (double)c;
    double b_squared = (double)b * (double)b;
    double y = (four_a_c - b_squared) / (4.0 * (double)a);
    jdouble_array_set(vertex, 0, x);
    jdouble_array_set(vertex, 1, y);
    return vertex;
}

File: FormalBench-data/FilteredData/selected_java/seed_726_per_category_10_653ade686f/ParabolaVertex.java

import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;

class ParabolaVertex {
    
    public static double[] parabolaVertex(int a, int b, int c) {
        double[] vertex = new double[2];
        vertex[0] = (-(double) b / (2.0 * a));
        vertex[1] = (((double) 4.0 * a * c) - ((double) b * b)) / (4.0 * a);
        return vertex;
    }
}

